web/openapi/openapi.yaml

```diff
@@ -153,7 +153,7 @@ paths:
   /export/with-schemas:
     post:
       summary: Export data with schema information
-      description: "Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/language) The data is returned grouped by schema.\n"
+      description: "Export data from VAST according to a query. The query must be a valid expression in the VAST Language. (see https://vast.io/docs/understand/language). The data is returned grouped by schema.\n"
```

Comment:
```suggestion
      description: "Export data from VAST according to a query. The query must be a valid expression in the VAST language. (see https://vast.io/docs/understand/language). The data is returned grouped by schema.\n"
```

Reply:
Replaced "VAST Language" with "VAST language" in https://github.com/tenzir/vast/pull/2959/commits/4b289bb23fce3c786bad9b58305f8910e4d0182a